### ISSUE TITLE
update information missing

### ISSUE THREAD

**Issue by AndKe** [guysoft/OctoPi#127]
Installed: 1.2.0-dev-939-g8e5c383
Available: 1.2.4 (stable) 

there is a box saying that only "v" marked updates can be installed "directly"
please provide some information about the updates that cannot.

**Comment by foosel** [comment]
https://github.com/foosel/OctoPrint/wiki/FAQ#octoprint-tells-me-i-have-an-update-available-but-i-cant-apply-it

**Comment by AndKe** [comment]
Thank you for responding.
Yes' I've read that prior to reporting.
This image is the "daily bulid" (I picked from the wrong end of http://docstech.net/OctoPiMirror/nightly/)

-The wiki page you linked to, leads me to https://github.com/foosel/OctoPrint/wiki/Plugin:-Software-Update#making-octoprint-updateable-on-existing-installations

Where I verified setup and config.yaml contents.
- Anyway . I'll proceed with getting proper nightly build from the end of the list, as this is most likely not a problem anymore.
